spec/openapi.yaml

```diff
@@ -2025,6 +2025,62 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/404response'
+    patch:
+      security:
+        - basic_auth: [ ]
+        - hmac_auth: [ ]
+      tags:
+        - Source Address
+      summary: Update My Own Number Label
+      description: |
+        Update label for my own number only.
+      operationId: updateSenderAddressUsingPATCH
+      parameters:
+        - name: id
+          in: path
+          description: The UUID of the sender address
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: 01ccaccb-e654-4503-bb7e-1acbcfc34cb2-XXXXXXXXXXX
```

Comment:
Why we have -XXXXXXXXXXX here? uuid format has just 5 part

Reply:
Updated